sensorlib/ReadMe.md

```diff
@@ -0,0 +1,88 @@
+# CARLA Sensor Library
+
+## Overview
+
+The sensor library provides a proxy sensor for the CARLA simulator, exposing additional features.
+
+This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a
```

Comment:
This sentence is probably slightly inaccurate? The feature is to enable object detection for our CDASim deployment using CARLA's existing sensor and object modeling ("object modeling" might not be the right word here either. What I am referring is CARLAs 3d world model that include dynamic vehicles and pedestrians and static obstacles).

Reply:
The main concern here is that this is not a CARLA-Driven simulation, we are just using CARLA for vehicle dynamics and now for 3d world modeling and object detection.